packages/core/src/components/tabs/_tabs.scss

```diff
@@ -6,28 +6,50 @@ $tabs-border-size: 1px;
 
 @include exports('ray-tabs') {
   .#{$ray-class-prefix}tabs {
-    border-bottom: $tabs-border-size solid $ray-color-blue-50;
-    display: flex;
+    &__list {
+      border-bottom: $tabs-border-size solid $ray-color-gray-10;
+      display: flex;
+    }
 
     &__item {
-      @extend .#{$ray-class-prefix}text--body-large;
+      font-size: 1rem;
+      padding-top: 0.75rem;
+      padding-bottom: 0.75rem;
       display: block;
       cursor: pointer;
       color: $ray-color-blue-50;
       text-align: center;
-      min-width: 200px;
-      padding-top: 1rem;
-      padding-bottom: 1rem;
       margin-bottom: -$tabs-border-size !important;
+      min-width: 85px;
```

Comment:
![image](https://user-images.githubusercontent.com/6812331/65162674-9cb14400-da07-11e9-950e-c3eac0299dca.png)

this solution doesn't work that well with labels >85px , as there then wont be any spacing between them

```suggestion
    padding-left: 0.5rem;
    padding-right: 0.5rem;
```

Reply:
could use `text-overflow` -- https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow